packages/components/HOW-TO-TEST-A-COMPONENT-IN-CLOUD-UI.md

```diff
@@ -2,13 +2,73 @@
 
 To test a component that is under development (or because it’s new or has been modified) you have a few options:
 
-- you can use your local environment as source for the `@hashicorp/design-system-components` package
 - you can install the `@hashicorp/design-system-components` package using a branch available on GitHub
+- ~you can use your local environment as source for the `@hashicorp/design-system-components` package~
 
 This is the case when the component has not yet been released as an npm package, of course.
 
+## Using a remote branch (via GitHub)
+
+> [!IMPORTANT]
+> While this approach is more suitable for one-off testing before releasing a package, less so for ongoing development, this remains the only option available for pre-testing our code in Cloud UI until we find an way to have the local environment approach work again.
+
+### 1 - Override the package resolution in the `package.json` file in the root
+
+You can override every dependency declared in the engines' `package.json` files with a single entry in the `package.json` file in the root of the project folder, by adding this line to the `"resolutions"` block:
+
+```
+"@hashicorp/design-system-components": "https://github.com/hashicorp/design-system#head=BRANCH-NAME&workspace=@hashicorp/design-system-components"
+```
+
+### 2 - Temporary commit the compiled folders
+
+Because the NPM package is released with the `dist` folder, but the monorepo ignores this folder via `.gitignore`, in your testing branch you will **temporarely** have to:
```

Comment:
```suggestion
Because the npm package is released with the `dist` folder, but the monorepo ignores this folder via `.gitignore`, in your testing branch you will **temporarily** have to:
```